fbw-common/src/wasm/systems/systems/src/communications/amu.rs

```diff
@@ -0,0 +1,861 @@
+                                mixed_audio.receive_com2 = reception != 0;
+                            } else if sdi == 3 {
+                                mixed_audio.volume_com3 = volume;
+                                // Comment due to vPilot that needs com3 to be received all the time
+                                //mixed_audio.receive_com3 = reception != 0;
+                            }
+                        }
+                        LabelWordAMUACP::Label213VolumeControlVORMKR => {
+                            if sdi == 1 {
+                                mixed_audio.volume_vor1 = volume;
+                                mixed_audio.receive_vor1 = reception != 0;
+                            } else if sdi == 2 {
+                                mixed_audio.volume_vor2 = volume;
+                                mixed_audio.receive_vor2 = reception != 0;
+                            } else if sdi == 3 {
+                                mixed_audio.receive_markers = reception != 0;
+                            }
+                        }
+                        LabelWordAMUACP::Label212VolumeControlADFPA => {
+                            // PA should be here but not simulated
+                            if sdi == 1 {
+                                mixed_audio.volume_adf1 = volume;
+                                mixed_audio.receive_adf1 = reception != 0;
+                            } else if sdi == 2 {
+                                mixed_audio.volume_adf2 = volume;
+                                mixed_audio.receive_adf2 = reception != 0;
+                            }
+                        }
+                        LabelWordAMUACP::Label217VolumeControlILS => {
+                            mixed_audio.volume_ils = volume;
+                            // TODO: Use data from future DMC. There's a wire between comms and DMC
+                            // FCOM compliant: ILS can be listened to only if LS is pressed
+                            mixed_audio.receive_ils = reception != 0;
+                        }
+                        _ => {}
+                    }
+                }
+            }
+        }
+
+        if can_send_amu_word {
+            AdaptationBoard::send_amu_word(bus, transmission_table_acp);
+        }
+    }
+
+    pub fn get_mixed_audio_acp(&self) -> MixedAudio {
+        self.mixed_audio_acp
+    }
+
+    pub fn get_mixed_audio_acp3(&self) -> MixedAudio {
+        self.mixed_audio_acp3
+    }
+
+    pub fn get_transmission_table_acp(&self) -> u32 {
+        self.transmission_table_acp
+    }
+
+    pub fn get_transmission_table_acp3(&self) -> u32 {
+        self.transmission_table_acp3
+    }
+}
+
+impl SimulationElement for AudioCard {
+    fn accept<T: SimulationElementVisitor>(&mut self, visitor: &mut T) {
+        self.acp.accept(visitor);
+
+        visitor.visit(self);
+    }
+}
+
+#[derive(Default)]
+pub struct Calls {
```

Comment:
```suggestion
struct Calls {
```
Also for the implemented functions.